jobs/postgres/spec

```diff
@@ -42,6 +44,9 @@ provides:
   - databases.tls.ca
 
 properties:
+  databases.version:
+      description: "The database version e.g. 11, 13, 15 or 16"
+      default: "16"
```

Comment:
indentation doesn't match the other properties

Reply:
Done